WordPress/Classes/Networking/TaxonomyServiceRemoteREST.m

```diff
@@ -2,76 +2,148 @@
 #import "WordPressComApi.h"
 #import "RemotePostCategory.h"
 #import "RemotePostTag.h"
+#import "RemoteTaxonomyPaging.h"
+
+static NSString * const TaxonomyServiceRemoteRESTCategoryTypeIdentifier = @"categories";
+static NSString * const TaxonomyServiceRemoteRESTTagTypeIdentifier = @"tags";
```

Comment:
So many letters :D 
Since these consts are static and won't be used outside this class maybe they could be shortened?